library/classes.lisp

```diff
@@ -7,7 +7,6 @@
    #:Signalable
    #:error
    #:Tuple
-   #:Optional #:Some #:None
```

Comment:
We might still want to import/re-export these symbols for backward compatibility.